### ISSUE TITLE
MIAF should clarify behaviour of multiple EXIFs and XMPs

### ISSUE THREAD

**Issue by leo-barnes** [MPEGGroup/FileFormat#60]
It's possible for an image item to reference multiple EXIF and XMP items. There are multiple closed-ecosystem use-cases where this might make sense:
- Storing the original EXIF as the second EXIF item when it has been modified for archival purposes
- Having multiple EXIF MakerNotes
- Storing different orthogonal pieces of custom metadata as separate XMP items. (iPhone images with depth maps stores multiple pieces of metadata for auxiliary images as separate XMP items)

But unless the parser has some kind of side information it's impossible for a parser to know what to do with them. XMP blobs can potentially be merged if they have different namespaces, but how do you merge two EXIF MakerNotes?

After discussing this a bit, I think it would make sense to add wording to MIAF that is similar to what [WebP](https://developers.google.com/speed/webp/docs/riff_container) does:
> There SHOULD be at most one chunk of each type ('EXIF' and 'XMP '). If there are more such chunks, readers MAY ignore all except the first one. Also, a file may possibly contain both 'EXIF' and 'XMP ' chunks.

Here's a relevant issue:
https://github.com/AOMediaCodec/av1-avif/issues/193

**Comment by cconcolato** [comment]
I wonder if relying on order of items is reliable. Maybe using item references is more appropriate? or adding a property to the old items?

**Comment by leo-barnes** [comment]
> Maybe using item references is more appropriate?

Right, that's what I had in mind, but I realize now upon re-reading that it's not at all clear it's what I meant.

**Comment by leo-barnes** [comment]
Accepted into AMD2
